aisynphys/database/schema/dataset.py

```diff
@@ -140,11 +142,17 @@ def stimulus_tseries(self):
         # ('first_spike', 'stim_spike.id', 'The ID of the first spike evoked by this pulse'),
         ('data', 'array', 'Numpy array of presynaptic recording sampled at '+sample_rate_str, {'deferred': True}),
         ('data_start_time', 'float', "Starting time of the data chunk, relative to the beginning of the recording"),
+        
+        ('position', 'object', '3D location of this stimulation in the arbitrary coordinate system of the experiment'),
+        ('device_name', 'str', 'The name of the device used for this stimulus'),
+        ('qc_pass', 'bool', 'Indicates whether this stimulation passed qc for testing a synaptic connection.')
```

Comment:
Say more about how you want to use this? StimPulse isn't exclusively intended for testing synaptic connections; it might be there's a better place for this column.

Reply:
I think my intention was for qc_pass to mean that the stimulation happened. For example in case there was a stimulation command, but the readout from a photodiode or other sensor didn't show a big enough increase. However, I don't think our data currently holds that information. 
I think I meant it to be analogous to the Spike qc_pass. But I don't remember that well and need to look closer to be sure

Reply:
Ok, so maybe document as `"Indicates whether this stimulation passed QC for generating a spike in the target cell (regardless of whether a spike actually occurred)"`